config/routes.rb

```diff
@@ -297,14 +297,15 @@
   get "/user/:display_name/account", :to => redirect(:path => "/account/edit")
   get "/user/:display_name/diary/comments(/:page)", :page => /[1-9][0-9]*/, :to => redirect(:path => "/user/%{display_name}/diary_comments")
 
-  resource :account, :only => [:edit, :update, :destroy] do
+  resource :account, :only => [:show, :update, :destroy] do
     scope :module => :accounts do
       resource :terms, :only => [:show, :update]
       resource :pd_declaration, :only => [:show, :create]
       resource :deletion, :only => :show
       resource :home, :only => :show
     end
   end
+  get "/account/edit", :to => redirect(:path => "/account"), :as => nil
```

Comment:
Do we actually need this redirect? Are we expecting sites outside openstreetmap.org to have direct links to the account edit page?

Reply:
This one is even less likely to be linked:
https://github.com/openstreetmap/openstreetmap-website/blob/483f4c1341ee2b0eb438dc7704e4ba272d1e74fe/config/routes.rb#L300

Do we need it?

Reply:
There seem to be a handful of hits on that but they may all be nonsense.